app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,63 @@
+class TeamsController < ApplicationController
+  before_action :set_team, only: [:show, :edit, :update, :destroy, :join, :leave]
```

Comment:
Maybe this should be inverted to an `except`? 🙈  WDYT?

Reply:
I'd go with this anyway, otherwise the reader would need to mentally compute the difference between the set of all implemented actions and the one(s) listed here.

The pattern seems to do it like this for this particular `before_action`, on Rails scaffolds, anyhow.